packages/cozy-procedures/src/locales/fr.json

```diff
@@ -21,6 +21,10 @@
   "personalDataForm": {
     "title": "Informations personnelles",
     "subtitle": "Complétez les informations suivantes pour finaliser votre dossier",
+    "completedStatus": {
+      "main": "%{completed} sur %{total} informations",
+      "rest": "ont déjà été complétées."
```

Comment:
We should probably pluralize this, but not blocking;

Reply:
did it, thanks!